geopandas/io/sql.py

```diff
@@ -306,7 +306,9 @@ def _psql_insert_copy(tbl, conn, keys, data_iter):
 
     dbapi_conn = conn.connection
     with dbapi_conn.cursor() as cur:
-        sql = "COPY {} ({}) FROM STDIN WITH CSV".format(tbl.table.fullname, columns)
+        sql = 'COPY "{}"."{}" ({}) FROM STDIN WITH CSV'.format(
```

Comment:
Sorry for a dumb question but what this `schema.name` syntax does? I thought that quotes are enough to fix the issue (that was the state of the PR when I reviewed it the first time). I am not a frequent user of PostgreSQL so I may miss something obvious.

Reply:
Oh sure, no question is dumb :) I too had assumed it was enough but the `to_sql` tests with schema names started failing which is when I realized that the `.` between `schemaname.tablename` cannot be inside quotes otherwise postgres will consider it as a part of the tablename. So now with this change, one can create a table with `.`  and other postgres wildcards which would have failed earlier, so technically one could create a table called `test.table.name` where test is the schema and table.name is the name of the table. 

Hope that clears things up

Reply:
Ah, cool. Thanks!